### DISCUSSION TITLE
No register on Ubport / Ubuntu Touch 20.04 | Version 2.0.4

### DISCUSSION THREAD

**Discussion by n00bsi** [axolotl-ai-cloud/axolotl#1975] [Q&A] [answered]
Hi,
anyone using axolotl on Ubuntu Touch ?

Scan the QR-Code to Register but nothing happen on the Ubuntu Touch.
Any idea how to get a log to see what happen on the Smartphone with Ubports/Ubuntu Touch 20.04 ?

Thanks

**Comment by ds2k5** [comment, marked as answer]
May you can contact the Developer of the APP: Aaron Kimming  https://github.com/axolotl-chat/axolotl

**Comment by n00bsi** [comment]
@ds2k5 

thank you will try
